### ISSUE TITLE
Demat. circuit  and new control suggestion by Alexer Zoderia

### ISSUE THREAD

**Issue by parar020100** [MattJeanes/TARDIS#584]
## (Quoted from Discord)

# TLDR
The throttle should have different functions based on whether flightmode or the demat circuit is engaged.
Throttle on + Demat engaged = Dematerialize
Throttle on + Flightmode engaged = Physical launch/Flightmode

Throttle off + Flightmode Engaged = Float
Throttle on while in vortex + Demat circuit/Lever disengaged = Rematerialize
Throttle off + Flight & Demat off = Landed
It can make that thud sound when the throttle engages too, and again when you shut the throttle off.
Basic Mode can make it to where hitting the throttle just does what it does now

# Full message:

Ok so I was having a thought on this right
Y'all guys are trying to make each TARDIS behave like it does in the series, and that's good and all, but there's a lot we don't know. 

Case in point, the Throttle Lever. 

I think it should have 2 functions, if possible:
The base function of it is Engine control. When activated by itself, it engages the engines but doesn't dematerialize or make the TARDIS take off initially. Those jobs are left to the Dematerialization Circuit and the Flight Circuit/Locking down mechanism, respectively.
Or alternatively, the Engine Release lever/control.
When the Dematerialization Circuit is engaged, activating the throttle will cause the TARDIS to dematerialize like it does now, and remain that way until the Dematerialization Circuit is disengaged again.
However, activating the Flightmode lever instead while the engines are engaged makes the TARDIS fly physically.
In either case, the Advanced mode can be reconfigured to start with either activating the Dematerialization Circuit or Flightmode control, rather than the manual or programmable destination controls
Alternatively you can, in an emergency takeoff situation, activate the Dematerialization/Flightmode circuit, engage the engine release, and then activate the throttle
This can, depending on if the doors are open/closed or the handbrake is engaged, cause significant damage to the TARDIS.
But also, therin lies the question of the Vortex Flight control.

This can be repurposed into the Dematerialization Circuit, and for "Short" hops, you can have a fallback set to a random coordinate on the map, or a saved set of coordinates in the programmable database.
If one is not chosen before takeoff, that is.
